base/src/test/java/com/mindera/skeletoid/webview/cookies/WebViewCookiesTest.kt

```diff
@@ -41,20 +37,24 @@ class WebViewCookiesTest {
     @Test
     fun testClearWebViewCookiesApiEqualOrGreaterThan22() {
         Whitebox.setInternalState(Build.VERSION::class.java, "SDK_INT", 27)
+        //Move to new ApplicationProvider.getApplicationContext() throws an error
+        //probably need to use https://stackoverflow.com/questions/32957741/android-illegalstateexception-no-instrumentation-registered-must-run-under-a-re
```

Comment:
Need to use what?

Reply:
`ApplicationProvider.getApplicationContext()` above

Reply:
Can we improve the comment?

```
// Moving to the new ApplicationProvider.getApplicationContext() throws an error
// We probably need to use (explain here what) like suggested in this SO thread
// https://stackoverflow.com/questions/32957741/android-illegalstateexception-no-instrumentation-registered-must-run-under-a-re
```